source/math/FloatMatrix.ooc

```diff
@@ -260,10 +260,8 @@ FloatMatrix : cover {
 				value := upper elements[y + y * upper width]
 				if (value != 0)
 					result elements[x + y * result width] = accumulator / value
-				else {
-					// TODO: What do we do about this?
-					/*DivisionByZeroException new() throw()*/
-				}
+				else
+					raise("Division by zero")
```

Comment:
`raise("Division by zero in FloatMatrix backwardSubstitution")` would be more helpful...